app/helpers/ui_constants.rb

```diff
@@ -492,6 +492,10 @@ module UiConstants
                       {:tables => ui_lookup(:tables => "miq_server")}
                     end,
   }
+  ASSIGN_TOS["MiddlewareServer"] = {
```

Comment:
The TOS is not a description. It is a class name, as a similar of other TOS descriptions.